### PR TITLE
Update glados_respeaker.py

### PR DESCRIPTION
**Pull request by eternalliving**
Fixed respeaker to honor settings.env

### CODE REVIEW

**Comment by eternalliving** [comment]
Trying to wrap my head around the intention of these lines when `respeaker_errors(e)` should really only be called if `os.getenv('RESPEAKER_CONNECTED')` is true.... There would then be no point for the extra check in the error handler.

**Comment by eternalliving** [comment]
When checking for boolean values in the settings.env they need to be checked against a list of possibles, or values. This is because dotenv strictly imports the values as strings.
https://stackoverflow.com/questions/63116419/evaluate-boolean-environment-variable-in-python

File: glados_respeaker.py [hunk 1]
```diff
@@ -19,13 +19,12 @@ def respeaker_errors(e):
 		exit();
 	if "name 'pixel_ring' is not defined" in str(e):
 		print("\nERROR: ReSpeaker is probably not connected?")
```
Comment (salleq) [inline glados_respeaker.py:21]: I think this line should be below the check for variable. It's pointless to mention ReSpeaker is not connected if it is disabled in config on purpose.

Comment (eternalliving) [reply glados_respeaker.py:21]: With the if statements fixed, the ones checking the settings.env, respeaker_errors() should never get called. Therefore this error mesase is still good if it’s enabled in config, but unplugged or not working. 

Also, I believe the variable checks in glados_functions.py are redundant, I’ll look into that more in a bit.

**Pull request merged**